README.md

```diff
@@ -152,25 +150,26 @@ If there isn't an issue for the work you want to do, please create one and inclu
 
 ## Create a branch
 
-It is preferable that you create a new branch with a name like `issue/##/<short-description>`. GitHub makes it pretty easy to associate branches and tickets, but it's just easier when it's in the name.
+It is preferable that you create a new branch with a name like `issue/##/<short-description>`. GitHub makes it pretty easy to associate branches and tickets, but it's nice when it's in the name.
 
 ## Testing the template
 
 Testing can be tricky. The current best way is to clone this repository locally, and use Copier to generate a test project locally, then verify your expected results.
 
-Copier will look for git tags to determine which version of the template to use. You probably don't want to create new tags while you're working on the template. Create a test project using this signature to let Copier know to use the latest local version. See the [Copier documentation](https://copier.readthedocs.io/en/latest/generating/#regenerating-a-project) for more information.
-
+Copier will look for git tags to determine which version of the template to use. You probably don't want to create new tags while you're working on the template. Create a test project using the following command to let Copier know to use the latest local version. 
 ```
 >> copier --vcs-ref HEAD </local/path/to/template> </test/project/directory>
 ```
 Notes:
 1) Any changes to the template will need to be committed (**not pushed**) to be picked up by Copier.
 2) If there's an opportunity for introducing an automated test, please take it.
+3) This project has some automated testing, to ensure that the template can render a reasonable output project - feel free to extend that if it's reasonable.
```

Comment:
Elsewhere, we call this "hydrate" (vs render). It would be nice to be consistent (since I think of render corresponding to rendering sphinx docs, which this template also helps with).